pkg/ee/resource-quota/handler.go

```diff
@@ -282,3 +295,50 @@ func DeleteResourceQuota(ctx context.Context, request interface{}, provider prov
 	}
 	return nil
 }
+
+func convertToAPIQuota(resourceDetails kubermaticv1.ResourceDetails) apiv2.Quota {
```

Comment:
I suggest adding some unit tests to cover cases like `cpu=500m` etc.

Reply:
Good idea, ill add some test cases.

Also about the `cpu=500m` we actually dont have anywhere support for cpu values under 1 in resource calculation. So that needs to be changed. I do that in a different PR.

Reply:
added some coversion to the current test case. Its not ideal, but to test better i would need to refactor the test to not have static resource quotas. Also I would like to avoid exporting this function just for tests. But feel free to change my mind, im not 100% about this.